doc/specs/application-auth-token.md

```diff
@@ -1,10 +1,25 @@
 ---
 description: >-
-  Version 0.0.1. The Pocket Network protocol contemplates the use of Application Auth Tokens to allow
-  Application Clients to access Service Nodes on behalf of the Application.
+  Version 0.0.1
+
+  The Pocket Network protocol contemplates the use of Application Auth Tokens (AATs)
+  to allow Gateways (off-chain) access Service Nodes (a.k.a Suppliers) on behalf
+  of the Application.
 ---
 
-# Application Authentication Token
+# Application Authentication Token <!-- omit in toc -->
+
+- [Data Structure Schema](#data-structure-schema)
```

Comment:
It'd be reader-friendly to have simple description about the purpose of AAT.  How about this?

---
The purpose of an AAT is to define two separate entities:

1. Application Owner to stake an application on the blockchain
2. Client Owner to submit relays through any type of dApp (from a command-line tool to Portal-like website)

The key point here is one entity doesn't have to share their private key with the other entity.  Application Owner can delegate the right to use their staked app to Client Owner by passing an AAT including the client public key signed by the application private key.

Reply:
Awesome. Ty!